shared/src/commonMain/kotlin/co/nimblehq/ic/kmm/suv/data/remote/body/SurveySubmissionApiBody.kt

```diff
@@ -0,0 +1,46 @@
+package co.nimblehq.ic.kmm.suv.data.remote.body
+
+import co.nimblehq.ic.kmm.suv.domain.model.AnswerSubmission
+import co.nimblehq.ic.kmm.suv.domain.model.QuestionSubmission
+import co.nimblehq.ic.kmm.suv.domain.model.SurveySubmission
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class SurveySubmissionApiBody(
+    @SerialName("survey_id")
+    val surveyID: String,
```

Comment:
same

```suggestion
    val surveyId: String,
```

Reply:
Fixed in https://github.com/suho/kmm-ic/pull/111/commits/4ce0598b700739a8d6220950c3bfe9fb2c432212